chart/values.yaml

```diff
@@ -196,6 +196,18 @@ mariadb:
     persistence:
       size: 1G
 
+elasticsearch:
+  enabled: true
+  #Choose versions in https://www.docker.elastic.co/ from "Elasticsearch" list
+  version: 6.5.0
+  persistence:
+    size: 512Mi
+    storageClassName: nfs
+  requests:
```

Comment:
We should pass in a whole `resources` parameter, so that we can add not only requests but also limits, this will be needed when we tune things, especially given how Elasticsearch tends to reserve any resources it can get its hands on.